test/inspect-test.js

```diff
@@ -96,6 +108,40 @@ tape('v8 inspect', (t) => {
         -1,
         'cons string content');
 
+    sess.send(`v8 inspect --string-length 20 ${cons}`);
+  });
+
+  sess.linesUntil(/">/, (lines) => {
+    lines = lines.join('\n');
+    t.notEqual(
+        lines.indexOf('this could be a bit ...'),
+        -1,
+        '--string-length truncates the string');
+
+    sess.send(`v8 inspect ${array}`);
+  });
+
+  sess.linesUntil(/}>/, (lines) => {
+    lines = lines.join('\n');
+    t.notEqual(
+        lines.indexOf('<Array: length=6'),
+        -1,
+        'array length');
+    t.ok(
+        lines.match(/\[5\]=0x[0-9a-f]+:<function: Class at .+\.js:\d+:\d+>}>$/),
```

Comment:
It doesn't look like you're using the result of `match()` here and on line 143. It might be better to use `test()` instead.